src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -14361,6 +14361,19 @@ public void PropertyInsert()
                 capabilities: EditAndContinueCapabilities.AddMethodToExistingType);
         }
 
+        [Fact]
+        public void Property_Insert_Static()
```

Comment:
Perhaps add a test for a static auto-prop, just for completeness?

Reply:
There is one: Property_Insert_Auto_Static

Reply:
Whoops, right you are. Not sure how I missed that earlier :)